HeterogeneousCore/CUDAUtilities/interface/AtomicPairCounter.h

```diff
@@ -0,0 +1,54 @@
+#ifndef HeterogeneousCoreCUDAUtilitiesAtomicPairCounter_H
+#define HeterogeneousCoreCUDAUtilitiesAtomicPairCounter_H
+
+#include<cstdint>
+
+class AtomicPairCounter {
+public:
+
+  using c_type = unsigned long long int;
+
+  AtomicPairCounter(){}
+  AtomicPairCounter(c_type i) { counter.ac=i;}
+
+#ifdef __CUDACC__
+  __device__ __host__
+  AtomicPairCounter & operator=(c_type i) { counter.ac=i; return *this;}
+#endif
+
+  struct Counters {
+    uint32_t n;  // total size
```

Comment:
can you use more meaningful variable names, e.g. `size` and `elements` ?

Reply:
ok